src/shared/metadata/cgroup_path_resolver.cc

```diff
@@ -26,20 +27,30 @@
 #include "src/common/fs/fs_wrapper.h"
 #include "src/shared/metadata/cgroup_path_resolver.h"
 
+#define S_MAGIC_CGROUP2FS 63677270
+DEFINE_bool(test_cgroup2_path, false, "Flag to force assume cgroup2 fs for testing purposes");
```

Comment:
naming: How about `force_cgroup2_mode`

Reply:
fixed in https://github.com/pixie-io/pixie/pull/635/commits/b976406be7fed6f63bebd138f6ba4a0a1021abf6